test/Kripke-v1.1/Kripke-v1.1-RAJA/Kripke/Timing.cpp

```diff
@@ -108,6 +114,9 @@ void Timing::stop(std::string const &name){
     HPM_Stop(name.c_str());
 #endif
 
+#ifdef RAJA_USE_CALIPER
+    cali::Annotation(name.c_str()).end();
+#endif
```

Comment:
Would it be better if we centralize on one timer class?  Or, maybe it won't matter since Kripke will be pulled out into a different repo.  Never  mind.